tests/spin-up-with-s3.sh

```diff
@@ -44,7 +44,7 @@ EXIT_CODE=$?
 set -e
 
 # cleanup
-kubectl delete ns $NAMESPACE
+kubectl delete ns $NAMESPACE || echo "Could not delete namespace ${NAMESPACE}"
```

Comment:
hm.. prob set the exit code so that way it tries to cleanup as much as it can, but then break?

Reply:
I just noticed that this fails if the namespace was never created, which means we then leak buckets.  Could also just move the `set -e` to after the cleanup...

Reply:
oh what i mean is something like:
```bash
kubectl delete ns $NAMESPACE || { echo "Could not delete namespace ${NAMESPACE}" ; EXIT_CODE=2; }
```

That way we know it's a failure to create the namespace, it'll still cleanup the bucket, but the test will fail because namespace failed.